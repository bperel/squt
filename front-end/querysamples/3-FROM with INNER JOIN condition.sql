SELECT comp.companyName, co.contractID AS ID 
FROM Contrat co 
INNER JOIN Company comp ON co.companyCode = comp.companyCode;