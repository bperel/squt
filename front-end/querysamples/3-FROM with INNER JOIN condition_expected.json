{
   "Tables" : {
      "contrat" : {
         "co" : {
            "OUTPUT" : {
               "contractID" : {
                  "-1" : "ID"
               }
            },
            "CONDITION" : {
               "companyCode" : {
                  "comp.companyCode" : "JOIN_TYPE_STRAIGHT"
               }
            }
         }
      },
      "company" : {
         "comp" : {
            "OUTPUT" : {
               "companyName" : {
                  "-1" : "companyName"
               }
            }
         }
      }
   }
}